mlx_graphs/transforms/normalize_features.py

```diff
@@ -0,0 +1,25 @@
+from typing import List, Union
+
+import mlx.core as mx
+
+from mlx_graphs.data import GraphData, GraphDataBatch
+from mlx_graphs.transforms import BaseTransform
+
+
+class NormalizeFeatures(BaseTransform):
+    def __init__(self, attributes: List[str] = ["node_features"]):
+        self.attributes = attributes
+
+    def process(
+        self, data: Union[GraphData, GraphDataBatch]
+    ) -> Union[GraphData, GraphDataBatch]:
+        if isinstance(data, GraphData):
+            for attribute in self.attributes:
+                array = getattr(data, attribute)
+                if array is not None:
+                    if array.size > 0:
+                        array = array - mx.min(array)
+                        sum_val = mx.sum(array, axis=-1, keepdims=True)
+                        new_array = array / mx.clip(sum_val, 1.0, None)
+                    setattr(data, attribute, new_array)
```

Comment:
What is this loop normalizing? How do we make sure only node/edge/graph features are normalized and not labels or other attributes?

Reply:
Its basically normalizing node features. I chose Elliptic bitcoin for this test (but it can work with any kind of features, node/edge). I did not find an optimized way to do this as mlx does not allow inplace division for some reason

Reply:
As specific attributes are passed into `attributes`, the user can easily control which attributes to normalize, so it shouldn't cause any problem regarding labels etc.